v2/go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/projectdiscovery/nuclei/v2
 
 go 1.19
 
+// replace github.com/projectdiscovery/gozero => C:\\Users\\marco\\go\\src\\github.com\\projectdiscovery\\gozero
+
```

Comment:
@Mzack9999 . i think this is not needed anymore right ?

Reply:
correct